src/core/resources/bivariateStatisticsResource/types.ts

```diff
@@ -7,7 +7,7 @@ import type {
   ColorCombination,
 } from '~utils/bivariate';
 
-type AxisDTO = Omit<Axis, 'id' | 'label'>;
+export type AxisDTO = Omit<Axis, 'id' | 'label'> & { label?: string | null };
```

Comment:
Does `label` property can be both `null` and `undefined`?

Reply:
You are right, this property doesn't need to be optional. Fixed to string | null